test/testutils.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+// disable eyeglass deprecations until tests are rewritten without deprecated methods
+process.env.EYEGLASS_DEPRECATIONS = "false";
```

Comment:
The reason I didn't do this currently is that it potentially complicates the options normalization phase. I'd have to normalize the options before I can check `disableDeprecations`, which would mean I'd have to queue up any deprecations encountered up to that point.

This would also mean I'd have to touch the tests in a more significant way, per my comment below.